src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -148,14 +264,20 @@
          :body
          {:columns
           (let [column-data (fn [ws]
-                              (let [disabled? (= (:accessLevel ws) "NO ACCESS")]
+                              (let [no-access? (= (:accessLevel ws) "NO ACCESS")
+                                    disabled? (and no-access? (= (get-in ws [:workspace :authorizationDomain :membersGroupName])
+                                                                 (config/dbgap-authorization-domain)))]
                                 {:workspace-id (select-keys (:workspace ws) [:namespace :name])
                                  :href (let [x (:workspace ws)] (str (:namespace x) ":" (:name x)))
```

Comment:
Since you're touching this file, this value appears to be unused, which is good because it's wrong.